.changeset/heavy-kings-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
+---
+
+Updated Block Alignment Styles for Deck
```

Comment:
How/why?

Also, is the title case intentional?

Reply:
Changesets just pulls in the title of the PR by default. I wasn't aware we were adding more details here or had a formatting convention. I'll copy the commit message in.

Reply:
The case isn't as big a deal as the detail. Sometimes the PR title is adequate on its own, but it might not always be. I've been following the guidance in [the changesets documentation](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-single-package-repository):

> While not every changeset is going to need a huge amount of detail, a good idea of what should be in a changeset is:
>
> - WHAT the change is
> - WHY the change was made
> - HOW a consumer should update their code

In this case I get the "what," I don't get the "why," and I'm not sure "how" is important.

I still consider myself a changeset novice, so stop me if I'm off base here!

Reply:
No, that makes sense (and I'm just as new as you are!).

I'll update my changeset again to focus more on the why, and I'll put this as a topic on the next Dev team meeting, because I don't think there's clarity on the team about what makes a good changeset message as opposed to commit message.